nodes/textures/iorpreset.py

```diff
@@ -0,0 +1,71 @@
+"""A BlendLuxCore node to provide index of refraction preset values for
+    LuxCoreRender in Blender"""
+# <pep8 compliant>
+import bpy
+from bpy.props import FloatProperty, StringProperty
+from .. import LuxCoreNodeTexture
+from ...operators import ior_presets
+
+
+class LuxCoreNodeIORPreset(LuxCoreNodeTexture):
+    """ Index of Refraction Preset node """
+    bl_label = "IOR Preset"
+    bl_width_min = 180
+    bl_width_default = 180
+
+    ior_name_text = StringProperty(name="IOR Name", description="The name of"
+                                   " the selected Index of Refraction preset")
+    ior_value_text = StringProperty(name="IOR Value", description="The value "
+                                    "of the selected Index of Refraction"
+                                    " preset")
+
+    def get_node_data_path(self):
+        return bpy.data.node_groups[self.id_data.name].nodes[self.name]
+
+    def update_ior_value_float(self, context):
+        self.get_node_data_path().outputs[0].default_value = \
```

Comment:
You can remove the `.get_node_data_path()` - `self` is already the node instance.

In fact, you can remove this whole line, because the default_value of the output is never used.
The default_value is only used on inputs without a node plugged in.

The place where you use self.ior_value_float is the export method below:
`"value": self.ior_value_float`